src/controllers/ScheduleController.js

```diff
@@ -682,6 +695,52 @@ async function clearGamesCollection() {
     }
 }
 
+router.get('/:id/move/up', isAdmin, async (req, res) => {
```

Comment:
nice functionality, but this has to come with some constraints (e.g. final cannot played before the semi-final etc.) but for now this is ok and we assume the user will break nothing - this is a fix after rewriting the scheduling algorithm

Reply:
Yes i thats right i thought about it when i added this but i couldnt find an elegant solution to implement all the required conditions so i implemented only that you cant switch with ended or active games - I hope the user takes responsibility for this